helm-charts/secrets-operator/values.yaml

```diff
@@ -22,6 +41,8 @@ controllerManager:
         cpu: 10m
         memory: 64Mi
   replicas: 1
+  serviceAccount:
```

Comment:
This doesn't seem to have anything to do with the PR, I assume we want to remove this part too?

Reply:
Also helmify auto generated